apps/web/src/pages/SkillFamilies/CreateSkillFamilyPage.tsx

```diff
@@ -103,7 +103,7 @@ export const CreateSkillFamilyForm = ({
 
   const skillOptions: Option<string>[] = sortedSkills.map(({ id, name }) => ({
     value: id,
-    label: name?.[locale] || "",
+    label: name?.[locale] ?? "",
```

Comment:
Pre-existing, I wonder if this should be a "not found" string instead?

Reply:
For all of these noted, I went and updated to use `getLocalizedName` which will fall back to not found.

[fix localized names null fallbacks](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11651/commits/37d38ad80f04bf3718d0ba3b6d072894c2fe03c1)